### PR TITLE
4.6 release notes

### PR DESCRIPTION
**Pull request by bmesuere**

### CODE REVIEW

File: en/news/2020-09-16-4-release-notes/README.md [hunk 1]
```diff
@@ -60,3 +60,9 @@ All details about this Dodona version can be found in [our GitHub release notes]
 Dodona 4.5 enables the grading of solutions for all users as a beta. In addition, the "all courses" page was redesigned to make it more useful. The courses of your own institution are now more prominent. In a future release a tab with "featured courses" will be added.
 
 All details about this Dodona version can be found in [our GitHub release notes](https://github.com/dodona-edu/dodona/releases/tag/4.5).
+
+## Dodona 4.6 - 2021-06-18
+
+Dodona 4.5 adds a "featured courses" section to the courses overview. The courses in this list contain instructions next to exercises and can be used for self-paced learning. As a teacher, they are also a great starting point for building your own course.
```
Comment (niknetniko) [inline en/news/2020-09-16-4-release-notes/README.md:66]:
```suggestion
Dodona 4.6 adds a "featured courses" section to the courses overview. The courses in this list contain instructions next to exercises and can be used for self-paced learning. As a teacher, they are also a great starting point for building your own course.
```

**Review by niknetniko — CHANGES_REQUESTED** [review]

File: en/news/2020-09-16-4-release-notes/README.md (hunk 1 above)
Comment (pdawyndt) [inline en/news/2020-09-16-4-release-notes/README.md:66]: 4.6

File: nl/news/2020-09-16-4-release-notes/README.md [hunk 2]
```diff
@@ -58,3 +58,9 @@ Alle details over deze Dodona versie kan je vinden in [onze GitHub release notes
 Dodona 4.5 schakelt de mogelijkheid om punten te geven tijdens een evaluatie in voor alle gebruikers. Deze feature is nog in beta, dus problemen mogen steeds gemeld worden. Daarnaast kreeg de overzichtspagina van alle beschikbare cursussen een update. Deze is nu gebruiksvriendelijker door een beter onderscheid te maken tussen de cursussen van de eigen onderwijsinstelling.
 
 Alle details over deze Dodona versie kan je vinden in [onze GitHub release notes](https://github.com/dodona-edu/dodona/releases/tag/4.5).
+
+## Dodona 4.6 - 18/06/2021
+
+Dodona 4.6 voegt "uitgelichte cursussen" toe aan het globale cursusoverzicht. Deze cursussen bevatten naast oefeningen ook instructies om een programmeerataal te leren. Voor leerkrachten zijn ze een goed startpunt om een eigen cursus op te bouwen.
```
Comment (chvp) [inline nl/news/2020-09-16-4-release-notes/README.md:64]:
```suggestion
Dodona 4.6 voegt "uitgelichte cursussen" toe aan het globale cursusoverzicht. Deze cursussen bevatten naast oefeningen ook instructies om een programmeertaal te leren. Voor leerkrachten zijn ze een goed startpunt om een eigen cursus op te bouwen.
```

File: en/news/2020-09-16-4-release-notes/README.md (hunk 1 above)
Comment (chvp) [inline en/news/2020-09-16-4-release-notes/README.md:66]:
```suggestion
Dodona 4.5 adds a "featured courses" section to the courses overview. The courses in this list contain instructions as well as exercises and can be used for self-paced learning. As a teacher, they are also a great starting point for building your own course.
```
or
```suggestion
Dodona 4.5 adds a "featured courses" section to the courses overview. The courses in this list contain instructions in addition to exercises and can be used for self-paced learning. As a teacher, they are also a great starting point for building your own course.
```
"next to" doesn't feel right to me, but I can't point to any grammar rules to support that.

**Review by chvp — APPROVED** [review]

**Pull request merged**